src/components/ItemGrid/ItemGrid.js

```diff
@@ -66,6 +67,18 @@ export class ItemGrid extends Component {
         this.props.acRemoveDashboardItem(clickedId);
     };
 
+    shouldComponentUpdate(nextProps, nextState) {
+        return (
+            !isEqual(
```

Comment:
How likely is the scenario where the number of items hasn't changed but the items are not the same?

Reply:
With current setup (only possible to add/remove one item at a time) I'd say it is impossible.